Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -1661,9 +1661,13 @@ AXCoreObject::AccessibilityChildrenVector AccessibilityRenderObject::documentLin
     AccessibilityChildrenVector result;
     Document& document = m_renderer->document();
     Ref<HTMLCollection> links = document.links();
+    WeakPtr cache = document.existingAXObjectCache();
```

Comment:
should we return early here instead of check !cache through each item in loop

Reply:
Yes! Will fix that.